aws-iot-thinggroup/aws-iot-thinggroup.json

```diff
@@ -58,7 +58,6 @@
       "properties": {
         "Attributes": {
           "type": "object",
-          "maxLength": 2800,
```

Comment:
wan't this validation done before as now it needs to be removed?

Reply:
The validation is done as part of `cfn validate` check. It was flagged as a warning - and not a blocking error.

Reply:
alright so now we are trying to remove the warning?